docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1220,6 +1220,16 @@ How to **investigate**
 - Flush tenant's data to blocks storage.
 - Remove tenant's directory on disk and restart ingester.
 
+## MimirStoreGatewayTooManyFailedOperations
+
+How it **works**:
+
+This alert fires when the `store-gateways` report errors when interacting with the object storage.
```

Comment:
Is there any "this is usually because of X" we can add here?

```suggestion
This alert fires when the `store-gateways` report errors when interacting with the object storage for an extended period of time.
```